.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: build
+on:
+  pull_request:
+  push:
+      branches:
+        - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
why not `fail-fast`? (just curious, I don't know much about minitest)

Reply:
https://www.edwardthomson.com/blog/github_actions_6_fail_fast_matrix_workflows.html

if we test more than one ruby, we want them all to finish so we can see results (vs stop after first one fails) as they run in parallel

Reply:
ooooh, very cool! TIL GHA matrices.